package.json

```diff
@@ -34,79 +34,80 @@
     "bluebird": "^3.5.3",
-    "bootstrap-sass": "^3.3.6",
-    "color": "^2.0.1",
+    "bootstrap-sass": "^3.4.1",
+    "color": "^3.1.0",
     "command-join": "^2.0.0",
     "d3": "^4.13.0",
-    "debug": "^3.1.0",
+    "debug": "^4.1.1",
     "electron-is-running-in-asar": "^1.0.0",
-    "etcher-sdk": "^1.3.11",
+    "etcher-sdk": "^1.3.2",
     "flexboxgrid": "^6.3.0",
     "immutable": "^3.8.1",
     "inactivity-timer": "^1.0.0",
     "lodash": "^4.17.10",
-    "mime-types": "^2.1.18",
-    "nan": "^2.9.2",
+    "mime-types": "^2.1.22",
+    "nan": "^2.13.1",
     "node-ipc": "^9.1.1",
     "path-is-inside": "^1.0.2",
-    "pretty-bytes": "^1.0.4",
-    "prop-types": "^15.5.9",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
-    "react2angular": "^4.0.2",
-    "redux": "^3.5.2",
-    "rendition": "4.41.1",
+    "pretty-bytes": "^5.1.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "react2angular": "^4.0.6",
+    "redux": "^4.0.1",
+    "rendition": "4.57.2",
     "request": "^2.81.0",
     "resin-corvus": "^2.0.1",
-    "roboto-fontface": "^0.9.0",
+    "roboto-fontface": "^0.10.0",
     "semver": "^5.1.1",
-    "styled-components": "^3.2.3",
-    "styled-system": "^3.1.11",
+    "styled-components": "^4.1.3",
+    "styled-system": "^4.0.5",
     "sudo-prompt": "^8.2.3",
     "uuid": "^3.0.1",
     "xml2js": "^0.4.17"
   },
   "devDependencies": {
-    "@babel/core": "^7.2.0",
+    "@babel/core": "^7.3.4",
     "@babel/plugin-proposal-function-bind": "^7.2.0",
-    "@babel/preset-env": "^7.2.0",
+    "@babel/preset-env": "^7.4.1",
     "@babel/preset-react": "^7.0.0",
-    "acorn": "^6.0.5",
-    "angular-mocks": "1.7.6",
+    "acorn": "^6.1.1",
+    "angular-mocks": "1.7.8",
     "babel-loader": "^8.0.4",
-    "chalk": "^1.1.3",
-    "electron": "3.1.3",
-    "electron-builder": "^20.38.5",
+    "chalk": "^2.4.2",
+    "electron": "4.1.1",
+    "electron-builder": "^20.39.0",
     "electron-mocha": "^6.0.4",
-    "eslint": "^4.17.0",
-    "eslint-config-standard": "^10.2.1",
+    "eslint": "^5.15.3",
+    "eslint-config-standard": "^11.0.0",
     "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-jsdoc": "^3.5.0",
-    "eslint-plugin-lodash": "^2.6.1",
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-jsdoc": "^4.4.1",
+    "eslint-plugin-lodash": "^5.1.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-promise": "^4.0.1",
     "eslint-plugin-react": "^7.11.1",
     "eslint-plugin-standard": "^3.0.1",
     "html-angular-validate": "^0.2.3",
     "html-loader": "^0.5.1",
-    "mocha": "^5.0.1",
+    "mocha": "^6.0.2",
     "mochainon": "^2.0.0",
-    "nock": "^9.2.3",
+    "nock": "^10.0.6",
     "node-gyp": "^3.8.0",
     "node-sass": "^4.7.2",
     "pkg": "^4.3.0",
     "sass-lint": "^1.12.1",
     "simple-progress-webpack-plugin": "^1.1.2",
     "spectron": "^5.0.0",
-    "webpack": "^4.27.0",
+    "versionist": "^4.1.1",
```

Comment:
You seem to be re-adding `versionist` when #2692 removed it.

Reply:
True. I'll remove it.